physics/MP/GFDL/v3_2022/gfdl_cloud_microphys_v3_mod.F90

```diff
@@ -0,0 +1,7377 @@
+!>\file gfdl_cloud_microphys_v3_mod.F90
+!! This file contains the entity of GFDL MP scheme Version 3.
+
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the FV3 dynamical core.
```

Comment:
It's a little weird to have this in a physics scheme in the CCPP, but I'm guessing that it's not our call.

Reply:
My guess is this is a relic from a prior code port. I guess we can leave it for now